admin/controllers/users/views/default/view.php

```diff
@@ -55,6 +57,84 @@
 			<button formaction='<?php echo Config::$uripath;?>/admin/users/action/delete' onclick='return window.confirm("Are you sure?")' class='button is-danger' type='submit'>Delete</button>
 		</div>
 	</h1>
+
+	<div id='content_search_controls' class='flex'>
+
+		<div class="field">
+			<label class="label">Search Name/Email</label>
+			<div class="control">
+				<input value="<?php echo $search; ?>" name="search" form="searchform" class="input" type="text" placeholder="">
+			</div>
+		</div>
+
+		<div class='field'>
+			<label class="label">State</label>
+			<div class='control'>
+				<div class="select">
+					<input type='hidden' name='filters[2][key]' value='state' form='searchform'/>
+					<select name="filters[2][value]" form="searchform">
+						<option value=''>State</option>
+						<option <?php if ($filters['state']==='1') { echo " selected "; }?> value='1'>Published</option>
+						<option <?php if ($filters['state']==='0') { echo " selected "; }?> value='0'>Unpublished</option>
+						<option <?php if ($filters['state']==='-1') { echo " selected "; }?> value='-1'>Deleted</option>
+					</select>
+				</div>
+			</div>
+		</div>
+
+		
+
+		<div class='field'>
+			<label class="label">Group</label>
+			<div class='control'>
+				<div class="select">
+					<select  name="groups[]" form="searchform">
+						<option value=''>Select Group</option>
+						<?php foreach ($all_groups as $g):?>
+							<option <?php if (in_array($g->id,$groups)) { echo " selected "; }?> value='<?=$g->id?>'><?=$g->display?></option>
+						<?php endforeach; ?>
+					</select>
+				</div>
+			</div>
+		</div>
+
+		<div class='field' id='content_search_tags_wrap'>
+			<label class="label">Tagged</label>
+			<div class='control'>
+				<div class="select">
+					<select id="content_search_tags" name="coretags[]" form="searchform" multiple>
+						<?php foreach ($applicable_tags as $t):?>
+							<option <?php if (in_array($t->id, $coretags)) { echo " selected "; }?> value='<?=$t->id?>'><?=$t->title?></option>
+						<?php endforeach; ?>
+					</select>
+				</div>
+			</div>
+		</div>
+		<script>
+		new SlimSelect({
+			select:'#content_search_tags'
+		});
```

Comment:
ill admit this is just me being nit picky at this point, so feel free to ignore, but in other projects, ive changed the slimselects default height, so that it matches bulma's since they are slightly off

Reply:
Not nitpicky - added tweak to default admin template.